perl.h

```diff
@@ -7557,13 +7557,10 @@ the plain locale pragma without a parameter (S<C<use locale>>) is in effect.
 #    define _CHECK_AND_OUTPUT_WIDE_LOCALE_UTF8_MSG(s, send)                 \
         STMT_START { /* Check if to warn before doing the conversion work */\
             if (! IN_UTF8_CTYPE_LOCALE && ckWARN(WARN_LOCALE)) {            \
-                UV cp = utf8_to_uvchr_buf((U8 *) (s), (U8 *) (send), NULL); \
+                UV cp = utf8_to_uv_or_die((U8 *) (s), (U8 *) (send), NULL); \
```

Comment:
casts away const when called in S_check_locale_boundary_crossing()

Reply:
Thanks